packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -69,10 +69,10 @@ export function SelectWalletModal({
     [],
   );
 
+  // eslint-disable-next-line @typescript-eslint/no-use-before-define
+  const listener = async () => onConnectUnisatWallet();
+
```

Comment:
If we remove the window.unisat.addListener calls from `onConnectUnisatWallet`, this should not be needed here